plugin/ochttp/server.go

```diff
@@ -153,39 +151,9 @@ type trackingResponseWriter struct {
 	writer     http.ResponseWriter
 }
 
-// Compile time assertions for widely used net/http interfaces
-var _ http.CloseNotifier = (*trackingResponseWriter)(nil)
-var _ http.Flusher = (*trackingResponseWriter)(nil)
-var _ http.Hijacker = (*trackingResponseWriter)(nil)
-var _ http.Pusher = (*trackingResponseWriter)(nil)
+// Compile time assertion for ResponseWriter interface
```

Comment:
I get that it's not possible to test this with a static assertion any longer but would it be possible to test with reflection?

Reply:
Feels like a who tests the tester kind of scenario. I think it makes more sense to test for actual scenarios (which we already have a few in) then some synthetic catch all test.